### PR TITLE
Added solved task-13-queue

### PR DESCRIPTION
**Pull request by SerivljaEvgen**

### CODE REVIEW

File: src/main/java/homework/task_13_queue/coffee/order/CoffeeOrderBoard.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+    //1
+    public void add(Order order) {
+        order.setOrderNum(nextOrderNumber);
+        queueOrderList.add(order);
+        nextOrderNumber++;
+    }
+
+    //2
+    public void deliver() {
```
Comment (kartius) [inline src/main/java/homework/task_13_queue/coffee/order/CoffeeOrderBoard.java]: You should return needed Order here

File: src/main/java/homework/task_13_queue/coffee/order/CoffeeOrderBoard.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        System.out.println("The closest in the order queue: \n" +
+                queueOrderList.poll());
+    }
+
+    //3
+    public void deliver(Order deliverOrder) {
```
Comment (kartius) [inline src/main/java/homework/task_13_queue/coffee/order/CoffeeOrderBoard.java]: The same here

File: src/main/java/homework/task_13_queue/coffee/order/CoffeeOrderBoard.java (hunk 1 above)
Comment (SerivljaEvgen) [reply src/main/java/homework/task_13_queue/coffee/order/CoffeeOrderBoard.java]: Updated solved task-13-queue (changed "deliver" methods)

File: src/main/java/homework/task_13_queue/coffee/order/CoffeeOrderBoard.java (hunk 2 above)
Comment (SerivljaEvgen) [reply src/main/java/homework/task_13_queue/coffee/order/CoffeeOrderBoard.java]: Updated solved task-13-queue (changed "deliver" methods)

**Review by kartius — APPROVED** [review]

**Pull request merged**